README.md

```diff
@@ -66,6 +66,24 @@ while not timeout.is_expired():
         t.reset()
 ```
 
+### Keep timeout even an app crash 
+
+You can set the optional attribute `persistent`
```

Comment:
We should explain how this future work with `reset` method.

Reply:
I will add a description in the next commit.